examples/cli/cli-client.rs

```diff
@@ -350,16 +419,20 @@ async fn create_client(cli: &Cli, account: IdentityStrategy) -> Result<Client, C
                 .await
                 .unwrap(),
         );
+        builder = builder.history_sync_url(MessageHistoryUrls::LOCAL_ADDRESS);
     } else {
         info!("Using dev network");
         builder = builder.api_client(
             ApiClient::create("https://grpc.dev.xmtp.network:443".into(), true)
                 .await
                 .unwrap(),
         );
+        builder = builder.history_sync_url(MessageHistoryUrls::DEV_ADDRESS);
```

Comment:
You should be able to chain these calls. The compiler won't let you?

Reply:
oh, i was just following what was already there. i'll adjust to use chaining!